modules/ROOT/pages/enable-inbound-traffic.adoc

```diff
@@ -110,8 +122,18 @@ If you do not see *Add TLS Context*, ensure you are viewing your secret group in
 . After saving your TLS context, click *Secret Groups*, then click the *Finish*. This enables TLS Context for your *Runtime Fabric.*
 
 == Enable Inbound Traffic
+The Runtime Fabric internal load balancer supports OpenSSL 1.1.1 and TLS 1.3, which provide the following security enhancements and performance improvements over TLS 1.2:
+ 
+* 2x or greater TLS 1.2 connection performance throughput boost versus prior Runtime Fabric internal load balancer running on OpenSSL 1.0.2.
```

Comment:
Don't need to repeat TLS 1.2 because it's in the intro:
```suggestion
* 2x or greater connection performance throughput boost versus a previous Runtime Fabric internal load balancer running on OpenSSL 1.0.2
```
Do we need all of "connection performance throughput boost"? I was really tempted to shorten it to "connection throughput," but was worried that that might change the meaning.